scripts/release-wrapper

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+if [ $# -ne 1 ]; then
+  echo "USAGE: $0 type"
+  exit 1
+fi
+
+TYPE=$1
+
+do_release () {
+  # TODO: add other release types
+  if [ "$TYPE" == "deb" ]; then
+    yarn run release $1
+    ./scripts/release-deb $1
+  fi
+}
+
+if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
@dickeyxxx pushed up f6f888f so that the branch could be passed in.  I also added `arch` to the args at this point since I run the `x86` and `x64` windows builds in parallel workflows.  In the future it might be good to split the deb release into two, or I could run the windows ones serially, just let me know if you want me to tweak anything at this point.